bpf/bpf_lxc.c

```diff
@@ -50,6 +50,20 @@ static __always_inline bool redirect_to_proxy(int verdict, __u8 dir)
 }
 #endif
 
+#ifdef ENABLE_CUSTOM_CALLS
+static __always_inline void
+encode_custom_prog_meta(struct __ctx_buff *ctx, __u32 ret, int direction,
+			int identity)
```

Comment:
nit: The types used here seem a little inconsistent, `ret` is typically a signed `int` in the parent functions while `identity` is typically a `__u32`.

Reply:
I was also wondering whether there's anything we should do about `ret` being both unsigned and > 8bits, to prevent possible mixups if someone unintentionally passed large or negative values in here.

Reply:
(I knew I wanted to double-check types somewhere before pushing!)
Sure, will fix. That's right, making sure the return value is between the expected bounds make sense, it's probably much safer to skip the tail call entirely if this is not the case. We won't be able to return the expected value otherwise.